controller/internal/routes/base_router.go

```diff
@@ -237,7 +172,7 @@ func CreateWithResponder(rc *response.RequestContext, rsp response.Responder, li
 			return
 		}
 
-		if sve, ok := err.(*schema.ValidationErrors); ok {
+		if sve, ok := err.(*apierror2.ValidationErrors); ok {
```

Comment:
apierror2 looks like a refactor collission

Reply:
Went through and fixed these